sklearn/linear_model/coordinate_descent.py

```diff
@@ -1084,7 +1084,7 @@ def fit(self, X, y):
             to avoid unnecessary memory duplication. If y is mono-output,
             X can be sparse.
 
-        y : array-like, shape (n_samples,) or (n_samples, n_targets)
+        y : array-like, shape (n_samples,)
```

Comment:
This is also used by MultiTask{Lasso,ElasticNet}CV so it is now more inaccurate than before!

Reply:
I see. Now I understand why {Lasso,ElasticNet}CV do not support multiple outputs. To fix this I can think of two ways: 
- different documentation for MultiTask/standard 
- different classes LinearModelCV, MultiTaskLinearModelCV so multiple outputs are also supported.

Reply:
@adrinjalali I just updated the doc with @adrinjalali suggestion. We can maybe open a new PR for extending multioutput to {Lasso,ElasticNet}CV